### PR TITLE
Selinux check port

### PR DESCRIPTION
**Pull request by vutkin**

### CODE REVIEW

File: tasks/install-mdex.yml [hunk 1]
```diff
@@ -3,11 +3,23 @@
...
+      register: installed_packages
+      until: installed_packages is succeeded
+
+    - name: "Install ansible selinux configure libraries"
+      package:
+        name: "{{ selibs_item }}"
+        state: present
+      register: installed_selinux_package
+      until: installed_selinux_package is succeeded
+      loop:
```
Comment (tgadiev) [inline tasks/install-mdex.yml]: Loop is not needed anymore for installing packages. See "Install requirements" task above as example.

File: tasks/install-mdex.yml [hunk 2]
```diff
@@ -75,6 +87,15 @@
```
Comment (tgadiev) [inline tasks/install-mdex.yml]: when can be much more simple:
```yaml
when: |
    mdex_version is version('6.5.2', '==') or
    mdex_version is version('11', '>=')
```

File: tasks/install-mdex.yml [hunk 3]
```diff
@@ -75,6 +87,15 @@
...
   block:
+
+    - name: "Enable connections to {{ mdex_port }} port"
+      seport:
+        ports: "{{ mdex_port }}"
+        proto: "tcp"
+        setype: "http_port_t"
+        state: "present"
+      when: ansible_selinux.status == "enabled" and ansible_selinux.mode != "disabled"
+
```
Comment (tgadiev) [inline tasks/install-mdex.yml]: 'when' conditionals can be a list in that case:
```yaml
when: 
  - ansible_selinux.status == "enabled"
  - ansible_selinux.mode != "disabled"
```

File: tasks/install-mdex.yml (hunk 1 above)
Comment (vutkin) [reply tasks/install-mdex.yml]: Fixed

File: tasks/install-mdex.yml (hunk 2 above)
Comment (vutkin) [reply tasks/install-mdex.yml]: Fixed

File: tasks/install-mdex.yml (hunk 3 above)
Comment (vutkin) [reply tasks/install-mdex.yml]: Does it make sense for 2 conditions?

Comment (tgadiev) [reply tasks/install-mdex.yml]: As discussed

Comment (vutkin) [reply tasks/install-mdex.yml]: Fixed

File: tasks/install-mdex.yml [hunk 4]
```diff
@@ -62,9 +59,7 @@
     creates: "{{ mdex_path }}"
   become: True
   when: |
-    mdex_version.split('.')[0] | int <= 6 and
-    mdex_version.split('.')[1] | int <= 5 and
-    mdex_version.split('.')[2] | int <= 1
+    mdex_version is version('6.5.2', '<=')
```
[the remark this answers is not in the document]
Comment (vutkin) [reply tasks/install-mdex.yml]: Fixed

**Pull request merged**